src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetController.kt

```diff
@@ -35,32 +36,34 @@ class WidgetController(
     )
 
     @PostMapping("/$WIDGET_START_IDENT_BTN_CLICKED")
-    fun handleStartIdentButtonClicked(@RequestParam(required = false, name = "hash") sessionHash: String?): ResponseEntity<String> {
+    fun handleStartIdentButtonClicked(@RequestParam(required = false, name = "hash") sessionHash: String?, @RequestHeader(HttpHeaders.USER_AGENT, required = false) userAgent: String?): ResponseEntity<String> {
```

Comment:
shouldn't we use a named argument? or the first argument is for the `name` ?

```suggestion
    fun handleStartIdentButtonClicked(@RequestParam(required = false, name = "hash") sessionHash: String?, @RequestHeader(name = HttpHeaders.USER_AGENT, required = false) userAgent: String?): ResponseEntity<String> {
```

Reply:
it works without it as well, but I will add the name.